src/uu/date/src/date.rs

```diff
@@ -244,6 +244,9 @@ pub fn uumain(args: impl uucore::Args) -> UResult<()> {
                 Ok(date) => {
                     // GNU `date` uses `%N` for nano seconds, however crate::chrono uses `%f`
                     let format_string = &format_string.replace("%N", "%f");
+                    if format_string.contains("%#z") {
+                        return Err(USimpleError::new(1, "do not use '%#z' is is undefined"));
```

Comment:
Another typo here. I think we can also be a bit more descriptive than the chrono error.